galata/test/jupyterlab/notebook-scroll.test.ts

```diff
@@ -440,43 +440,50 @@ test.describe('Jump to execution button', () => {
     await page.notebook.openByPath(`${tmpPath}/${longOutputsNb}`);
     await page.notebook.activate(longOutputsNb);
   });
-  test('should show jump button after first execution and scroll to executing cells', async ({
-    page
-  }) => {
-    await page.notebook.setCell(0, 'code', 'from time import sleep\nsleep(2)');
-
-    // Button doesn't exist before execution
-    const indicator = page.locator('.jp-Notebook-ExecutionIndicator');
-    await indicator.hover();
-    await expect(
-      page.locator('.jp-Notebook-ExecutionIndicator-jumpButton')
-    ).toHaveCount(0);
 
-    // Run first cell
-    void page.notebook.runCell(0, false);
+  test.fixme(
+    'should show jump button after first execution and scroll to executing cells',
```

Comment:
Was this failing? I would not add `test.fixme` to tests which are just flaky <50% of time because then when a regression happens, we won't get a signal.

> test.fixme() marks the test as failing. Playwright will not run this test, as opposed to the fail annotation